### PR TITLE
Fix tests

### PR DESCRIPTION
**Pull request by Reallife**
| Q             | A
| ------------- | ---
| Bug fix?      | Yes or No
| New feature?  | Yes or No

### Added

- Dev dependency `avtocod/specs`
- Dependency `ocramius/package-versions`
- Method `Avtocod\Specifications\Specifications::getVendorDirectoryPath()`

### Changed 

- Namespaces for sdk classes

### Removed

- Redundant tests that check specifications files (validation, duplicates, directory structures etc)
- Dependency `opis/json-schema`

## Checklist

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I wrote unit tests for my code _(if tests is required for my changes)_
- [x] I have made changes in `CHANGELOG.md` file

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -18,23 +18,25 @@
...
-        "avtocod/specs": "~3.0",
+        "ext-mbstring": "*",
         "illuminate/support": "^5.6 || ~6.0 || ~7.0",
-        "tarampampam/wrappers-php": "^1.5"
+        "ocramius/package-versions": "^1.4",
+        "tarampampam/wrappers-php": "^1.5",
+        "mindplay/composer-locator": "^2.1"
     },
     "require-dev": {
+        "avtocod/specs": "~3.24",
```
Comment (tarampampam) [inline composer.json]: Maybe move in `require`?

File: src/Specifications.php [hunk 2]
```diff
@@ -2,23 +2,28 @@
...
+use Avtocod\Specifications\Structures\VehicleMark;
+use Avtocod\Specifications\Structures\VehicleType;
+use Avtocod\Specifications\Structures\VehicleModel;
+use Avtocod\Specifications\Structures\IdentifierType;
 use Tarampampam\Wrappers\Exceptions\JsonEncodeDecodeException;
 
 class Specifications
 {
+    /**
+     * Self package name.
```
Comment (tarampampam) [inline src/Specifications.php]:
```suggestion
     * Specifivations data package name.
```

File: src/Specifications.php [hunk 3]
```diff
@@ -29,6 +34,40 @@ class Specifications
      */
     public const VEHICLE_TYPE_DEFAULT = 'ID_TYPE_CAR';
 
+    /**
+     * Get current package version.
```
Comment (tarampampam) [inline src/Specifications.php]:
```suggestion
     * Get specifications data package version.
```

File: src/Specifications.php [hunk 4]
```diff
@@ -29,6 +34,40 @@ class Specifications
...
+    /**
+     * Get the specifications root directory path.
+     *
+     * @param string|null $additional_path
+     *
+     * @return string
+     */
+    public static function getRootDirectoryPath(string $additional_path = null): string
+    {
+        $root = \ComposerLocator::getPath(static::AVTOCOD_SPECS_PACKAGE_NAME);
```
Comment (tarampampam) [inline src/Specifications.php]: Or use reflection + `. self::AVTOCOD_SPECS_PACKAGE_NAME`?

**Review by tarampampam — CHANGES_REQUESTED** [review]

File: src/Specifications.php (hunk 4 above)
Comment (Reallife) [reply src/Specifications.php]: Made in [db8f1c0](https://github.com/avtocod/specs-php/pull/1/commits/db8f1c0c2f6ea09f5469543d43c5b98dcc98a2b2)

**Comment by tarampampam** [comment]
@Reallife uptade PR description, please

**Comment by tarampampam** [comment]
LGTM :+1:

**Review by tarampampam — APPROVED** [review]

**Comment by Reallife** [comment]
> @Reallife uptade PR description, please

Done

**Pull request merged**